src/app/layout/qgslayoutmanagerdialog.cpp

```diff
@@ -286,7 +288,14 @@ void QgsLayoutManagerDialog::mTemplate_currentIndexChanged( int indx )
 
 void QgsLayoutManagerDialog::mTemplatesDefaultDirBtn_pressed()
 {
-  openLocalDirectory( mDefaultTemplatesDir );
+  if ( QDir( mDefaultTemplatesDir ).exists() )
```

Comment:
I think this logic should be flipped - use the user-defined path first, and fallback to the (useless?) bundled path after.

Reply:
Since the current (pre 3.8) behaviour is the bundled path but that will not exist in 98.943% [*] of all cases, it's safe to keep it as default, because where it exists it does on purpose. And where it does not exist users will not even be notified about the legacy behaviour.

In short: it doesn't hurt anyone but protects systems which build on this functionality.

[*] All numbers have officially been verified by the FNPA (Fake News Protection Agency)